src/rpc/net.cpp

```diff
@@ -101,6 +101,19 @@ UniValue getpeerinfo(const UniValue& params, bool fHelp)
             "    \"lastrecv\": ttt,                      (numeric) the time in seconds since epoch (Jan 1 1970 GMT) of the last receive\n"
             "    \"bytessent\": n,                       (numeric) the total bytes sent\n"
             "    \"bytesrecv\": n,                       (numeric) the total bytes received\n"
+            "    \"bytessent_per_msg\":  {\n"
+            "        msg,                                (string) the total bytes sent aggregated by message type.\n"
```

Comment:
Not sure here the fields `\"bytessent_per_msg\"` and `\"bytesrecv_per_msg\"` are properly documented, in my opinion some more details on the fact it's a dictionary of keys as strings and values as numerics should be provided.